src/server/ApiRoutes/Bib.js

```diff
@@ -4,11 +4,13 @@ import nyplApiClient from '../routes/nyplApiClient';
 import logger from '../../../logger';
 import appConfig from '../../app/data/appConfig';
 import extractFeatures from '../../app/utils/extractFeatures';
+import { itemBatchSize } from '../../app/data/constants';
 
-const nyplApiClientCall = (query, urlEnabledFeatures) => {
+const nyplApiClientCall = (query, urlEnabledFeatures, itemFrom) => {
```

Comment:
What value does `itemFrom` take?

Reply:
`itemFrom` should be a whole number (denoting the index of the first item, i.e. if `itemFrom` is 3000, should only return hits listed after the first 3000 items). This is used for pagination.

Reply:
Why not call it `offset`?

Reply:
*`item_offset`. But not a big deal. Since it seems thats what we went with on the backend already.

Reply:
Yes `itemFrom` is just to be consistent with the backend `item_from` parameter.